lib/pr_checker.js

```diff
@@ -183,14 +185,13 @@ export default class PRChecker {
         r.user.login !== pr.author.login &&
         collaborators.includes(r.user.login.toLowerCase())).length;
 
-      if (requester === pr.author.login && approvals < 2) {
-        cli.error('The fast-track request requires' +
-                  " at least two collaborators' approvals (👍).");
-        return false;
-      } else if (approvals === 0) {
-        cli.error('The fast-track request requires' +
-                  " at least one collaborator's approval (👍).");
-        return false;
+      const missingFastTrackApprovals = FAST_TRACK_MIN_APPROVALS - approvals -
+        (requester === pr.author.login ? 0 : 1);
+      if (missingFastTrackApprovals > 0) {
+        isFastTracked = false;
+        fastTrackAppendix = ' (or 0 minutes if there is ' +
```

Comment:
Sorry for my previous suggestion, it should probably be hours rather than minutes, to align with the other messages.

```suggestion
        fastTrackAppendix = ' (or 0 hours if there is ' +
```